example/new-editor-example.js

```diff
@@ -1,5 +1,5 @@
 import wescheme from '../src/languages/wescheme';
-import CodeMirrorBlocks from '../src/CodeMirrorBlocks';
+import {CodeMirrorBlocks} from '../src/CodeMirrorBlocks';
```

Comment:
Since you updated `src/CodeMirrorBlocks` to match `new-reactify`, this import needs to be `import CodeMirrorBlocks` rather than `import {CodeMirrorBlocks}`. Same thing in `spec/new-block-test`.